src/views/Dashboard.tsx

```diff
@@ -1,11 +1,36 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import axios from 'axios';
+
 import DashboardHeader from '../components/DashboardHeader';
 import SideBar from '../components/sidebar/SideBar';
 import DashboardList from '../components/dashboard/dashboard-list/DashboardList';
 
 import './Dashboard.scss';
 
 const Dashboard: React.FunctionComponent = () => {
+    const [applicantList, setApplicantList] = useState([{ name: 'Placeholder', role: 'Placeholder' }]);
+    const [didFetchList, setDidFetchList] = useState(false);
+
+    useEffect(() => {
+        const applicants: any[] = [];
+        axios
+            .get('http://localhost:4000/applicant', { timeout: 2000 })
+            .then((result: any) => {
+                result.data.forEach((applicant: any) => {
+                    applicants.push({
+                        name: `${applicant.firstName} ${applicant.lastName}`,
+                        role: applicant.role,
+                    });
+                });
+                setApplicantList(applicants);
+                setDidFetchList(true);
+                console.log(applicantList);
+            })
+            .catch((err) => {
+                console.log(err);
+            });
+    }, [didFetchList]);
```

Comment:
Remove dependency array altogether because we only need to run the component once